metadata.yaml

```diff
@@ -0,0 +1 @@
+src/metadata.yaml
```

Comment:
If this is a reactive charm, you don't need to have a `metadata.yaml` file in the root directory. Only under `src/` so you could remove this symlink.